src/cursor.rs

```diff
@@ -72,6 +72,66 @@ impl Command for MoveTo {
     }
 }
 
+/// A command that moves the terminal cursor up the given number of lines, and resets it to the first column at the beginning of the line.
```

Comment:
* Contributing guidelines violation - line is too long
* _resets it to the first column at the beginning of the line_ - sounds like the same thing is said twice. What about just `and moves it to the first column.`?